ui/components/multichain/create-eth-account/create-eth-account.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import {
+  addNewAccount,
+  setAccountLabel,
+  getNextAvailableAccountName as getNextAvailableAccountNameFromController,
+} from '../../../store/actions';
+import { CreateAccount } from '..';
+
+export const CreateEthAccount = ({ onActionComplete }) => {
+  const dispatch = useDispatch();
+
+  const onCreateAccount = async (name) => {
+    const newAccountAddress = await dispatch(addNewAccount());
+    if (name) {
+      dispatch(setAccountLabel(newAccountAddress, name));
+    }
+    onActionComplete(true);
+  };
+
+  const getNextAvailableAccountName = async (_accounts) => {
```

Comment:
do we need `_accounts` if its not used?

Reply:
It was part of my initial implementation, and I tried to keep this "generic" (like if we have to add a new `CreateAccount` that wants to compute its name based on the existing account list).

Since it's a JS file, I guess we could remove it, and that would be ok, but I'd like to keep passing it on the `CreateAccount` component if that's fine for you?

Reply:
I did remove it!